operations/workplace/people/README.md

```diff
@@ -30,14 +31,14 @@ Description: >-
   * [💬](https://emojipedia.org/speech-balloon/)[People Ops Channel](https://community.mattermost.com/private-core/channels/people-ops)
   * 📁[Legacy PeopleOps Page](https://docs.mattermost.com/process/people-ops.html)\(to be [tomb-stoned](../../../company/about-mattermost/list-of-terms.md#tomb-stoned)\)
 * Compensation
-  * 🧑@natalie.jew - [Natalie Jew](https://mattermost.bamboohr.com/employees/employee.php?id=174&page=2077) - Director of HR  or 🧑@lynn.conway - [Lynn Conway](https://mattermost.bamboohr.com/employees/employee.php?id=207&page=2077)
+  * @idayat.ogungbade - [Idayat Ogungbade](https://mattermost.bamboohr.com/employees/employee.php?id=323&page=2077) - Total Rewards Manager 
```

Comment:
@lynn915, I've checked these URLs and they default to my profile which leads me to think only the person whose profile it is can see those details. Would you be open to me replacing the link with the link to Mattermost? That's what we use for other instances of this type of information.